perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -62,17 +62,41 @@ def Run(benchmark_spec):
     A list of sample.Sample objects.
   """
   vms = benchmark_spec.vms
-  if not vms[0].IsReachable(vms[1]):
-    logging.warn('%s is not reachable from %s', vms[1], vms[0])
+  results = []
+  for origin_index in [0, 1]:
+    origin_vm = vms[origin_index]
+    receiving_vm = vms[origin_index ^ 1]
+    results = results + _RunPing(origin_vm,
+                                 receiving_vm,
+                                 receiving_vm.internal_ip,
```

Comment:
While you are at it how about adding support for external ips as well?

There is code in iperf you can follow... https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_benchmarks/iperf_benchmark.py#L70

    if vm_util.ShouldRunOnExternalIpAddress():
      vm.AllowPort(IPERF_PORT)